@navikt/core/css/form/combobox.css

```diff
@@ -120,6 +120,14 @@
   padding-left: 0.25rem;
 }
 
+.navds-combobox__selected-options[data-type="single"] {
+  gap: 0;
```

Comment:
I think we should consider only adding the gap on multiselect instead of first adding it and then removing it.

Also, we _could_ use an existing class instead of `data-type`, unless this becomes "too clever":
```css
.navds-combobox__selected-options:not(:has(.navds-combobox__selected-options--no-bg)) {
  gap: var(--__ac-combobox-wrapper-inner-padding);
}
```

Reply:
Changed to "adding" it instead if removing it 🤔 Im a fan of relatively "dumb" selectors that makes fewer assumptions on different elements if possible. (In this case we have to assume that single-select options get the class `navds-combobox__selected-options--no-bg` when in "single-select" mode)